brave-lists/debounce.json

```diff
@@ -47,6 +47,15 @@
     "action": "redirect",
     "param": "Url"
   },
+  {
+    "include": [
+      "*://trk.pm-srv.co/*"
```

Comment:
I would make this a bit tighter to ensure we only debounce the links we know:
```
"*://trk.pm-srv.co/v2/trk?*"
```

Reply:
For example, if there's a `/v3/` later, it may not use the `rurl` param the same way.

Reply:
sound sensible